lib/terraforming/resource/iam_group_policy.rb

```diff
@@ -16,22 +16,22 @@ def initialize(client)
       end
 
       def tf
-        apply_template(@client, "tf/iam_group_policy")
+        apply_template(@client, 'tf/iam_group_policy')
       end
 
       def tfstate
         iam_group_policies.inject({}) do |resources, policy|
           attributes = {
-            "group" => policy.group_name,
-            "id" => iam_group_policy_id_of(policy),
-            "name" => policy.policy_name,
-            "policy" => prettify_policy(policy.policy_document, breakline: true, unescape: true)
+            'group' => policy.group_name,
+            'id' => iam_group_policy_id_of(policy),
+            'name' => policy.policy_name,
+            'policy' => prettify_policy(policy.policy_document, breakline: true, unescape: true)
```

Comment:
__[rubocop]__
* Line is too long. [96/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)